Remove outdated FAQ reference to keptn install on Quickstart page (#1424)
keptn/keptn.github.io: content/docs/quickstart/_index.md

## FAQ

### Disk pressure on pods in Kubernetes

> If the installation of Keptn is timing out, you can check if the root cause are low resources, such as disk space, by executing `kubectl describe pod PODID -n keptn`.

Reviewer: ```suggestion
If the installation of Keptn is timing out, you can whether this is caused by low resources, such as disk space, by executing `kubectl describe pod PODID -n keptn`.
```
The "root cause are low resources" construct is problematic -- I think it should be "is" rather than "are" to match the singular "root cause" but I suggest rewording to just eliminate the problem.
